docs/proposals/impersonate/impersonate-user.md

```diff
@@ -0,0 +1,115 @@
+    ```bash
+    kubectl create -f ./deploy
+    ```
+5. Create a ServiceInstance as the admin who has all privileges. Run: 
+    ```bash
+   kubectl create -f ./example/si_admin.yaml --as=admin@kyma.cx
+    ```
+6. Check the status of the created ServiceInstance. Run:
+    ```bash
+    kubectl get serviceinstance redis-admin
+    ```   
+   The ServiceInstance should be in a **Ready** state. Example output:
+   ```bash
+   NAME             CLASS                       PLAN      STATUS   AGE
+   redis-admin      ClusterServiceClass/redis   default   Ready    10s
+   ```
+
+7. Create a ServiceInstance as **user2** which doesn't have permission for creating K8s resources other that ServiceInstance. Run: 
+    ```bash
+   kubectl create -f ./example/si_user2.yaml --as=user2@kyma.cx
+    ```
+8. Check the status of the created ServiceInstance. Run:
+    ```bash
+    kubectl get serviceinstance redis-user2
+    ```   
+   The ServiceInstance should be in a **OrphanMitigationSuccessful** state. Example output:
+   ```bash
+   NAME          CLASS                       PLAN      STATUS                       AGE
+   redis-user2   ClusterServiceClass/redis   default   OrphanMitigationSuccessful   13m
+   ```
+
+   You can check the Service Catalog controller logs to find the provisioning error. Run:
+   ```bash
+    kubectl logs $(kubectl get po -n kyma-system -l app=service-catalog-catalog-controller-manager --no-headers -ocustom-columns=name:.metadata.name) -n kyma-system -c controller-manager | grep "Error syncing ServiceInstance impersonate-helm-broker/redis-user2"
+    ```
+   Expected output: 
+   ```bash
+    I0615 19:07:13.243344       1 controller.go:404] Error syncing ServiceInstance impersonate-helm-broker/redis-user2 (retry: 0/15): Error provisioning ServiceInstance of ClusterServiceClass (K8S: "123-123-123-123-123-123" ExternalName: "redis") at ClusterServiceBroker "impersonate-helm-broker": Status: 500; ErrorMessage: <nil>; Description: namespaces is forbidden: User "user2@kyma.cx" cannot create resource "namespaces" in API group "" at the cluster scope; ResponseError: <nil>
+   ```
+
+### Consequences for an accepted proposal
+
+The PoC shows that the [OriginatingIdentity](https://github.com/kubernetes-sigs/service-catalog/blob/b6afbc9fec94e7b0d350f22e736d0484d181b351/pkg/features/features.go#L31-L37) feature is not working as expected. When ServiceInstance is created by the user then the given ServiceInstance is processed by the Service Catalog controller. The controller adds fields under the `spec` entry causing that it is the last one that modified the ServiceInstance and the UserInfo is always populated with the Service Catalog controller service account. This breaks the flow because the Helm Broker does not know the actual author. Before implementing the proposed solution in Helm Broker the pull request which fixes that issue needs to be merged and a new Service Catalog needs to be used in Kyma.
```

Comment:
minor:
```
The PoC shows that the [OriginatingIdentity](https://github.com/kubernetes-sigs/service-catalog/blob/b6afbc9fec94e7b0d350f22e736d0484d181b351/pkg/features/features.go#L31-L37) feature is not working as expected.
```
Maybe we could show that more transparent e.g by showing some output when Instance created by some user has controller-manager userInfo? We could add it under point 8 in above section.

Reply:
We can also link to SC issue here if it does exist